2017-09-24 23:46:50,476 - log.py[line:38] - INFO: 测试开始时间：2017-09-24_23_46_50
2017-09-24 23:46:50,476 - log.py[line:92] - INFO:   run 生成测试用例脚本.
2017-09-24 23:46:52,539 - log.py[line:94] - INFO:   生成测试用例脚本 end.
2017-09-24 23:46:52,668 - log.py[line:92] - INFO:   run 读取待测设备信息.
2017-09-24 23:46:52,688 - log.py[line:94] - INFO:   读取待测设备信息 end.
2017-09-24 23:46:52,690 - log.py[line:92] - INFO:   run 初始化driver.
2017-09-24 23:46:53,711 - remote_connection.py[line:478] - DEBUG: POST http://127.0.0.1:4723/wd/hub/session {"capabilities": {"alwaysMatch": {"platformName": "Android"}, "firstMatch": [{}]}, "desiredCapabilities": {"deviceName": "sdk", "unicodeKeyboard": true, "app": "D:\\project\\Nidone_python\\com.hongdao.mamainst_24.apk", "resetKeyboard": true, "platformVersion": "4.3", "platformName": "Android"}}
2017-09-24 23:47:33,615 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-24 23:47:33,615 - log.py[line:94] - INFO:   初始化driver end.
2017-09-24 23:47:33,642 - log.py[line:92] - INFO:   run 正确登录.
2017-09-24 23:47:33,642 - remote_connection.py[line:478] - DEBUG: POST http://127.0.0.1:4723/wd/hub/session/d3f00805-f3ee-4aa4-ba61-f68ab9a48d05/context {"sessionId": "d3f00805-f3ee-4aa4-ba61-f68ab9a48d05", "name": "WEBVIEW"}
2017-09-24 23:47:33,822 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-24 23:47:33,822 - log.py[line:103] - ERROR: Message: No such context found.

2017-09-24 23:47:33,823 - mkdir_log_directory.py[line:30] - INFO: D:\project\Nidone_python\log\2017-09-24_23_46_50_error目录创建成功
2017-09-24 23:47:33,823 - remote_connection.py[line:478] - DEBUG: GET http://127.0.0.1:4723/wd/hub/session/d3f00805-f3ee-4aa4-ba61-f68ab9a48d05/screenshot {"sessionId": "d3f00805-f3ee-4aa4-ba61-f68ab9a48d05"}
2017-09-24 23:47:35,203 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-24 23:47:35,227 - operation.py[line:82] - INFO: 错误截图已保存在 D:\project\Nidone_python\log\2017-09-24_23_46_50_error
2017-09-24 23:47:35,229 - log.py[line:85] - ERROR: 'ascii' codec can't decode byte 0xe6 in position 50: ordinal not in range(128)
2017-09-24 23:47:35,230 - log.py[line:94] - INFO:   正确登录 end.
2017-09-24 23:47:35,230 - remote_connection.py[line:478] - DEBUG: DELETE http://127.0.0.1:4723/wd/hub/session/d3f00805-f3ee-4aa4-ba61-f68ab9a48d05 {"sessionId": "d3f00805-f3ee-4aa4-ba61-f68ab9a48d05"}
2017-09-24 23:47:42,851 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-24 23:47:42,851 - log.py[line:92] - INFO:   run 初始化driver.
2017-09-24 23:47:43,871 - remote_connection.py[line:478] - DEBUG: POST http://127.0.0.1:4723/wd/hub/session {"capabilities": {"alwaysMatch": {"platformName": "Android"}, "firstMatch": [{}]}, "desiredCapabilities": {"deviceName": "sdk", "unicodeKeyboard": true, "app": "D:\\project\\Nidone_python\\com.hongdao.mamainst_24.apk", "resetKeyboard": true, "platformVersion": "4.3", "platformName": "Android"}}
2017-09-24 23:48:23,286 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-24 23:48:23,288 - log.py[line:94] - INFO:   初始化driver end.
2017-09-24 23:48:23,311 - log.py[line:92] - INFO:   run 正常登录1.
2017-09-24 23:48:23,312 - remote_connection.py[line:478] - DEBUG: POST http://127.0.0.1:4723/wd/hub/session/0e84ead4-29cd-4b5b-8dd0-12432d21d40f/element {"using": "xpath", "sessionId": "0e84ead4-29cd-4b5b-8dd0-12432d21d40f", "value": "//*[@id=\"loginForm\"]/div[1]/div[1]/div[2]/input"}
2017-09-24 23:48:27,578 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-24 23:48:27,578 - log.py[line:103] - ERROR: Message: An element could not be located on the page using the given search parameters.

2017-09-24 23:48:27,578 - remote_connection.py[line:478] - DEBUG: GET http://127.0.0.1:4723/wd/hub/session/0e84ead4-29cd-4b5b-8dd0-12432d21d40f/screenshot {"sessionId": "0e84ead4-29cd-4b5b-8dd0-12432d21d40f"}
2017-09-24 23:48:29,033 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-24 23:48:29,035 - operation.py[line:82] - INFO: 错误截图已保存在 D:\project\Nidone_python\log\2017-09-24_23_46_50_error
2017-09-24 23:48:29,036 - log.py[line:85] - ERROR: 'ascii' codec can't decode byte 0xe6 in position 50: ordinal not in range(128)
2017-09-24 23:48:29,036 - log.py[line:94] - INFO:   正常登录1 end.
2017-09-24 23:48:29,036 - remote_connection.py[line:478] - DEBUG: DELETE http://127.0.0.1:4723/wd/hub/session/0e84ead4-29cd-4b5b-8dd0-12432d21d40f {"sessionId": "0e84ead4-29cd-4b5b-8dd0-12432d21d40f"}
2017-09-24 23:48:36,792 - remote_connection.py[line:565] - DEBUG: Finished Request
